statistics/handle/update.go

```diff
@@ -283,7 +283,7 @@ func (h *Handle) DumpStatsDeltaToKV(dumpMode bool) error {
 	h.sweepList()
 	currentTime := time.Now()
 	for id, item := range h.globalMap {
-		if dumpMode == DumpDelta && !needDumpStatsDelta(h, id, item, currentTime) {
+		if !dumpMode && !needDumpStatsDelta(h, id, item, currentTime) {
```

Comment:
I think we should assign a type for `dumpMode` instead of using `bool`.

Reply:
I think so, maybe I will add it tonight.

Reply:
added.